stable/aws-for-fluent-bit/values.yaml

```diff
@@ -4,7 +4,7 @@ global:
 
 image:
   repository: public.ecr.aws/aws-observability/aws-for-fluent-bit
-  tag: 2.21.5
+  tag: 2.31.2
```

Comment:
Currently we have issues in our latest versions: https://github.com/aws/aws-for-fluent-bit/issues/542

Which we believe may be fixed in 2.31.4+

For now, it might be safest to keep this chart on our stable: https://github.com/aws/aws-for-fluent-bit#using-the-stable-tag

Which is 2.28.4

Reply:
Done